assets/state-vgpu-device-manager/0600_daemonset.yaml

```diff
@@ -45,15 +45,29 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: LD_PRELOAD
+          value: "/driver-root/usr/lib64/libnvidia-ml.so.1"
```

Comment:
Ideally we wouldn't set this in the spec. Instead, we would discover the path to `libnvidia-ml.so.1` in the vgpu-device-manager code and set the `LD_PRELOAD` envvar when invoking the mig-parted binaries.  We have precedent for this in other components. For example, here is our code in the `gpu-operator-validator` that validates a driver container installation: https://github.com/NVIDIA/gpu-operator/blob/6324d2aca562edf46d93cbf9d2a0837ab5c12e59/cmd/nvidia-validator/main.go#L731-L749